### DISCUSSION TITLE
Propagation Mode

### DISCUSSION THREAD

**Discussion by Michael-AUG** [magicbug/Cloudlog#2680] [Q&A]
Hi everyone,

I am trying to make some slight amendments to Tobias' `cloudlog rigctld interface`.

For some reason, whenever the interface uploads data to CloudLog, it defaults to SATELLITE in the propagation mode.

What value can I send to the API to tell CloudLog that there is no special prop_mode being used? I have tried `0`, `" "` and `""` but, while these may work for one upload, CloudLog then defaults back to Satellite.

Many thanks
Michael GM5AUG

**Comment by magicbug** [comment]
Cloudlog expects the following style payload, anything not required completely remove.

```json
{
    "key": "YOUR_API_KEY",          // API-Key
    "radio": "QO-100 Station",      // Name of the radio (used for assigning received data)
    "frequency": "2400170000",      // Frequency in Hz
    "mode": "SSB",                  // Mode
    "frequency_rx": "10489670000",  // Optional Rx frequency in Hz
    "mode_rx": "SSB",               // Optional Rx mode (not logged)
    "prop_mode": "SAT",             // Optional propagation mode
    "sat_name": "QO-100",           // Optional satellite name
    "power": "5",                   // Optional transmit power in Watts
}
```

**Reply by Michael-AUG** [reply]
Hi.
Thanks for this.
I've removed the non-required fields but Cloudlog is still seeming to default to Satellite.
Even when I try using `CloudlogBashCat` I have the same problem.
Is there any chance there is something inside Cloudlog that is defaulting to Satellite, ignoring what the interfaces are sending?
Ta
M

**Reply by magicbug** [reply]
Don't believe so, just tested with CloudlogCAT here and its not defaulting to satellite when I select the rig control.

Other thing might be to delete the radio from the hardware interface page and just in case its left over weirdness

**Reply by Michael-AUG** [reply]
Amazing - deleting the radio from the hardware interface page seems to have done the trick. Thank you :)

**Reply by MarcDBehr** [reply]
I am seeing the same thing.  In rigctlCloudlogInterface.php, if I send prop_mode as "" or " " or if I don't send a "prop_mode" at all, when I go to the Live QSO page, it shows the mode as "Satellite". I removed the rig from the hardware page, restarted the client several times, and it still always defaults to Satellite propagation mode.

Ant other suggestions?
